src/github_runner_image_builder/builder.py

```diff
@@ -86,90 +82,147 @@
 MOUNTED_YQ_BIN_PATH = IMAGE_MOUNT_DIR / "usr/bin/yq"
 IMAGE_DEFAULT_APT_PACKAGES = [
     "docker.io",
+    "gh",
+    "jq",
     "npm",
     "python3-pip",
+    "python-is-python3",
     "shellcheck",
-    "jq",
-    "wget",
     "unzip",
-    "gh",
+    "wget",
 ]
 
 
+def initialize() -> None:
+    """Configure the host machine to build images.
+
+    Raises:
+        BuilderSetupError: If there was an error setting up the host device for building images.
+    """
+    try:
+        logger.info("Installing dependencies.")
+        _install_dependencies()
+        logger.info("Enabling network block device.")
+        _enable_network_block_device()
+    except ImageBuilderBaseError as exc:
+        raise BuilderSetupError from exc
+
+
 def _install_dependencies() -> None:
     """Install required dependencies to run qemu image build.
 
     Raises:
         DependencyInstallError: If there was an error installing apt packages.
     """
     try:
-        subprocess.run(
-            ["/usr/bin/apt-get", "update", "-y"], check=True, timeout=30 * 60
+        subprocess.check_output(
```

Comment:
Do you think for these and others that we should log the output?

Reply:
I was wondering if that subprocess run logs are necessary given that we log what is running for each step (e.g. "installing dependencies"). If there is an error, we will know via logger.exception at the end of the try/except calls. What do you think?

Reply:
Those logs are super useful for debugging